.github/workflows/link-check-deploy.yml

```diff
@@ -12,9 +12,10 @@ jobs:
       github.event.deployment.ref != 'main' &&
       github.event.deployment_status.state == 'success'
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
I'll follow up with a change to update `actions/checkout` in all our other actions as well, from what I've seen it's a no-downsides performance upgrade.